src/components/community/Platforms/types.ts

```diff
@@ -0,0 +1,37 @@
+import type { Platforms as PlatformsType } from "temporaryData/types"
+
```

Comment:
I feel like it's better to put everything that's only used at one component to that component's file at the top. We should have a discussion about it